src/app/dataset-block/metadata-block/components/event-details/components/common/block-interval-property/block-interval-property.component.html

```diff
@@ -1,17 +1,20 @@
-<p>
-    start:&nbsp;
-    <app-display-hash
-        [value]="data.block.start"
-        [navigationTargetDataset]="datasetInfo"
-        [showCopyButton]="true"
-    ></app-display-hash>
-</p>
-<br />
-<p>
-    end:&nbsp;
-    <app-display-hash
-        [value]="data.block.end"
-        [navigationTargetDataset]="datasetInfo"
-        [showCopyButton]="true"
-    ></app-display-hash>
-</p>
+<ng-container *ngIf="data.block; else noBlock">
+    <p>
+        start:&nbsp;
+        <app-display-hash
+            [value]="data.block.start"
+            [navigationTargetDataset]="datasetInfo"
+            [showCopyButton]="true"
+        ></app-display-hash>
+    </p>
+    <br />
+    <p>
+        end:&nbsp;
+        <app-display-hash
+            [value]="data.block.end"
+            [navigationTargetDataset]="datasetInfo"
+            [showCopyButton]="true"
+        ></app-display-hash>
+    </p>
+</ng-container>
+<ng-template #noBlock> - </ng-template>
```

Comment:
I don't think these changes are related to scheduling topic. Are they?

Reply:
Yes, I noticed a small bug and decided to fix it right away